docs/pages/eas/metadata/schema.mdx

```diff
@@ -269,6 +283,39 @@ EAS Metadata uses the least restrictive answer for each of these questions by de
   ]}
 </MetadataTable>
 
+#### Apple advisory age rating override
+
+<MetadataTable headers={['Name', 'Description']}>
+  {[
+    { name: 'NONE', description: 'No age rating override' },
+    {
+      name: 'SEVENTEEN_PLUS',
+      description: 'App contains content that may not be suitable for children under the age of 17',
+    },
+    {
+      name: 'UNRATED',
+      description: 'Adults Only. This content can\'t be published on the App Store. It may be published on alternative app marketplaces on iOS or websites in the European Union',
```

Comment:
```suggestion
      description: 'Adults Only. This content cannot be published on the App Store. It may be published on alternative app marketplaces on iOS or websites in the European Union.',
```